src/components/Auth/AuthConfig.js

```diff
@@ -0,0 +1,173 @@
+      uiDef.buttonColorPressed
+    defaultCustomization.guidanceCustomization.buttonTextNormalColor =
+      uiDef.buttonTextColor
+    defaultCustomization.guidanceCustomization.buttonTextDisabledColor =
+      uiDef.buttonTextColor
+    defaultCustomization.guidanceCustomization.buttonTextHighlightColor =
+      uiDef.buttonTextColor
+    defaultCustomization.guidanceCustomization.buttonRelativeWidth = '1f'
+    defaultCustomization.guidanceCustomization.buttonCornerRadius =
+      uiDef.buttonCornerRadius
+    defaultCustomization.guidanceCustomization.retryScreenImageBorderColor = color(
+      'neutral-white'
+    )
+    defaultCustomization.guidanceCustomization.retryScreenImageBorderWidth =
+      '2px'
+    defaultCustomization.guidanceCustomization.retryScreenImageCornerRadius =
+      '0'
+    defaultCustomization.guidanceCustomization.retryScreenOvalStrokeColor =
+      uiDef.retryScreenOvalColor
+    // defaultCustomization.guidanceCustomization.retryScreenSlideshowImages = retryScreenSlideshowImages
+    defaultCustomization.guidanceCustomization.retryScreenSlideshowInterval =
+      '1500'
+    defaultCustomization.guidanceCustomization.enableRetryScreenSlideshowShuffle = true
+    defaultCustomization.guidanceCustomization.enableRetryScreenBulletedInstructions = true
+
+    // Set Oval Customization
+    defaultCustomization.ovalCustomization.strokeColor = uiDef.ovalColor
+    defaultCustomization.ovalCustomization.progressColor1 =
+      uiDef.dualSpinnerColor
+    defaultCustomization.ovalCustomization.progressColor2 =
+      uiDef.dualSpinnerColor
+
+    // Set Feedback Customization
+    defaultCustomization.feedbackCustomization.backgroundColor =
+      uiDef.feedbackBarColor
+    defaultCustomization.feedbackCustomization.textColor =
+      uiDef.feedbackBarTextColor
+    defaultCustomization.feedbackCustomization.cornerRadius =
+      uiDef.buttonCornerRadius
+    defaultCustomization.feedbackCustomization.relativeWidth = '1f'
+
+    // Set Result Screen Customization
+    defaultCustomization.resultScreenCustomization.backgroundColors =
+      uiDef.frameColor
+    defaultCustomization.resultScreenCustomization.foregroundColor =
+      uiDef.textColor
+    defaultCustomization.resultScreenCustomization.activityIndicatorColor =
+      uiDef.accentColor
+    defaultCustomization.resultScreenCustomization.resultAnimationBackgroundColor =
+      uiDef.accentColor
+    defaultCustomization.resultScreenCustomization.resultAnimationForegroundColor =
+      uiDef.frameColor
+    defaultCustomization.resultScreenCustomization.uploadProgressFillColor =
+      uiDef.accentColor
+
+    this.currentCustomization = defaultCustomization
+
+    // Set Animated Customization
+    const loaderSVG = document.createElementNS(
+      'http://www.w3.org/2000/svg',
+      'svg'
+    )
+    loaderSVG.setAttribute('viewBox', '0 0 56 56')
+    loaderSVG.classList.add('experiment-svg')
+    loaderSVG.innerHTML = loader
```

Comment:
Using innerHTML in the client can be a security concern. Please find an alternative or make sure the html is properly sanitised.

Reply:
I've tried with multiple different ways to get the `SVGElement` to be accepted/rendered, but nothing except this works. This is also the method used by them on their examples, so I'm begrudgingly suspecting that it really is the only way... is there any recommendation you can provide to further sanitise this piece of code?